tools/test/examples/examples_lib.py

```diff
@@ -19,7 +19,8 @@
 from tools.export import EXPORTERS
 
 SUPPORTED_TOOLCHAINS = ["ARM", "IAR", "GCC_ARM"]
-SUPPORTED_IDES = ["iar", "uvision", "make_gcc_arm", "make_iar", "make_armc5"]
+SUPPORTED_IDES = [exp for exp in EXPORTERS.keys() if exp != "cmsis" and exp != "zip"]
```

Comment:
Is there a reason why "cmsis" and "zip" exporters aren't being listed in the `supported_ides`? Do they lack build steps?